packages/bluesnap-direct-integration/src/bluesnap-direct-credit-card-payment-strategy.ts

```diff
@@ -0,0 +1,26 @@
+import {
+    PaymentIntegrationService,
+    PaymentStrategy,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+export default class BlueSnapDirectCreditCardPaymentStrategy implements PaymentStrategy {
+    constructor(private _paymentIntegrationService: PaymentIntegrationService) {}
+
+    execute(): Promise<void> {
```

Comment:
Hey @josesonora can you please add the rest of strategy as well. Can do it in a separate commit. Since most of the code is here already and feel will be better to move it in one chunk.

Reply:
Hey @animesh1987, the code is mine 😅. I added the rest of the code.